spec/catalog.yaml

```diff
@@ -59,4 +59,191 @@ paths:
       operationId: Ping
       responses:
         '201':
-          description: OK
\ No newline at end of file
+          description: OK
+
+  /v1/namespaces:
+    post:
+      tags: [ Namespace ]
+      summary: >
+        Create a new namespace. A catalog can manage one or more namespaces. A namespace is used to manage one or more tables.
+        There are three modes when trying to create a namespace:
+          * CREATE: Create the namespace if it does not exist. If a namespace of the same name already exists, the operation fails with 400.
+          * EXIST_OK: Create the namespace if it does not exist. If a namespace of the same name already exists, the operation succeeds and the existing namespace is kept.
+          * OVERWRITE: Create the namespace if it does not exist. If a namespace of the same name already exists, the existing namespace is dropped and a new namespace with this name with no table is created.
+      operationId: CreateNamespace
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateNamespaceRequest'
+      responses:
+        200:
+          $ref: '#/components/responses/CreateNamespaceResponse'
+        400:
+          $ref: '#/components/responses/BadRequestErrorResponse'
+        403:
+          $ref: '#/components/responses/ForbiddenResponse'
+        406:
+          $ref: '#/components/responses/UnsupportedOperationResponse'
+        503:
+          $ref: '#/components/responses/ServiceUnavailableResponse'
+        5XX:
+          $ref: '#/components/responses/ServerErrorResponse'
+
+components:
+
+  schemas:
+    ErrorModel:
+      type: object
+      description: JSON error payload returned in a response with further details on the error
+      required:
+        - type
+        - title
+        - detail
+        - instance
+      properties:
+        type:
+          type: string
+          description: a URI identifier that categorizes the error
+          example: /errors/incorrect-user-pass
+        title:
+          type: string
+          description: a brief, human-readable message about the error
+          example: Incorrect username or password
+        status:
+          type: integer
+          minimum: 400
+          maximum: 600
+          description: HTTP response code
```

Comment:
nit: we should say this field (if present) must match the actual HTTP code returned by the service